src/registrar/admin.py

```diff
@@ -1061,6 +1079,37 @@ def organization_type(self, obj):
 
     organization_type.admin_order_field = "domain_info__organization_type"  # type: ignore
 
+    def federal_agency(self, obj):
+        return obj.domain_info.federal_agency if obj.domain_info else None
+
+    federal_agency.admin_order_field = "domain_info__federal_agency"  # type: ignore
+
+    def federal_type(self, obj):
+        return obj.domain_info.federal_type if obj.domain_info else None
+
+    federal_type.admin_order_field = "domain_info__federal_type"  # type: ignore
+
+    def organization_name(self, obj):
+        return obj.domain_info.organization_name if obj.domain_info else None
+
+    organization_name.admin_order_field = "domain_info__organization_name"  # type: ignore
+
+    def is_election_board(self, obj):
+        return obj.domain_info.is_election_board if obj.domain_info else False
+
+    is_election_board.admin_order_field = "domain_info__is_election_board"  # type: ignore
+    is_election_board.short_description = "Election office"  # type: ignore
+
+    def city(self, obj):
+        return obj.domain_info.city if obj.domain_info else None
+
+    city.admin_order_field = "domain_info__city"  # type: ignore
+
+    def state_territory(self, obj):
+        return obj.domain_info.state_territory if obj.domain_info else None
+
+    state_territory.admin_order_field = "domain_info__state_territory"  # type: ignore
```

Comment:
I wrote a mixin, `OrderableFieldsMixin`, to automate all of this code generation. This mixin will also allow you to do sorting on multiple fields at once. All you need to do is to define a field called "orderable_fk_fields" (could probably be a better name). Each field is specified as a tuple: (field, order_by). Order_by can either be a single value or a list depending on what you need.

The only thing it does not current support is a custom short_description but that would be fairly easy to pass in - it would just be a modifcation here if a value exists
![image](https://github.com/cisagov/manage.get.gov/assets/141044360/b2443d8f-c002-4370-b4bc-8d833b760c04)


![image](https://github.com/cisagov/manage.get.gov/assets/141044360/5adc9c02-a70d-47c3-858d-edb516d0f478)